include/bluetooth/mesh/cfg_cli.h

```diff
@@ -273,6 +273,20 @@ int bt_mesh_cfg_app_key_add(uint16_t net_idx, uint16_t addr, uint16_t key_net_id
 int bt_mesh_cfg_app_key_get(uint16_t net_idx, uint16_t addr, uint16_t key_net_idx,
 			    uint8_t *status, uint16_t *keys, size_t *key_cnt);
 
+
+/** @brief Delete an application key from the target node.
+ *
+ *  @param net_idx     Network index to encrypt with.
+ *  @param addr        Target node address.
+ *  @param key_net_idx Network key index the application key belongs to.
+ *  @param key_app_idx Application key index.
+ *  @param status      Status response parameter.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_cfg_app_key_del(u16_t net_idx, u16_t addr, u16_t key_net_idx,
+			    u16_t key_app_idx, u8_t *status);
```

Comment:
The Zephyr integer types have been deprecated, could you please convert these to stdint's `uint16_t` and `uint8_t`? Same throughout